pygotham/frontend/talks.py

```diff
@@ -100,6 +100,13 @@ def proposal(pk=None):
         db.session.add(talk)
         db.session.commit()
 
+        speaker = Speaker(
```

Comment:
That is a fantastic point, do we know how pygotham.org /  other conferences handle this? I think we should let all of the speakers edit talk information. If you invite someone to give a talk with you, then you likely trust them to modify the content of said talk.